src/world/World.php

```diff
@@ -588,6 +602,193 @@ private function initRandomTickBlocksFromConfig(ServerConfigGroup $cfg) : void{
 		}
 	}
 
+	/**
+	 * @return "clear"|"rain"|"thunder"
```

Comment:
When I said get rid of the enum, I didn't mean replace it with bare strings...
I meant to get/set the rain & thunder level directly, and use the rain and thunder level to calculate what the weather status is in /weather. The weather status isn't needed anywhere except /weather.